guides/source/getting_started.md

```diff
@@ -1204,10 +1204,10 @@ it look as follows:
 This time we point the form to the `update` action, which is not defined yet
 but will be very soon.
 
-Passing the article object to the method, will automagically create url for submitting the edited article form.
-This option tells Rails that we want this form to be submitted
-via the `PATCH` HTTP method which is the HTTP method you're expected to use to
-**update** resources according to the REST protocol.
+Passing the `@article` object to the method will automatically set the URL for
```

Comment:
I think I'd prefer to keep "the article object" here -- it's the class that affects the URL, but mentioning the variable name here seems to risk implying it's relevant. What do you think?

Reply:
Good call.  After re-reading the rest of the guide, just the plain "article object" text makes sense here.  The specific syntax of `model: @article` is described below anyway.

I'll revert it and push up a new commit.